docs/cs/fluttertodostutorial.md

```diff
@@ -320,7 +320,7 @@ class TodosBloc extends Bloc<TodosEvent, TodosState> {
   TodosBloc({@required this.todosRepository});
 
   @override
-  TodosState get initialState => TodosLoading();
+  TodosState get initialState => TodosLoadInProgress();
 
   @override
   Stream<TodosState> mapEventToState(TodosEvent event) async* {
```

Comment:
don't we have to update the actual body of `mapEventToState` to use the updated names?

Reply:
sorry i just change tutorial not doc